src/components/beacon/beacon.tsx

```diff
@@ -11,23 +11,40 @@ import { CommonProps } from '../common';
 import classNames from 'classnames';
 
 import { euiBeaconStyles } from './beacon.styles';
+import { useEuiTheme } from '../../services';
+
+export const COLORS = [
+  'primary',
+  'success',
+  'accent',
+  'danger',
+  'warning',
+] as const;
+
+export type EuiBeaconColor = typeof COLORS[number];
 
 export type EuiBeaconProps = Omit<HTMLAttributes<HTMLDivElement>, 'children'> &
```

Comment:
> the `color` prop doesn't show up in the documentation site. Any other prop name works fine except for color.

@dawitam11, the reason for that is because the type `HTMLAttributes<HTMLDivElement>` contains a `color` prop too. So you need to omit the color prop coming from this type.
 
```suggestion
export type EuiBeaconProps = Omit<
  HTMLAttributes<HTMLDivElement>,
  'children' | 'color'
> &
```

Reply:
@miukimiu Thank you! It works now.